apps/archive/archive.py

```diff
@@ -870,8 +870,6 @@ def deschedule_item(self, updates, original):
         :param original: original is document.
         """
         updates[ITEM_STATE] = CONTENT_STATE.PROGRESS
-        updates[PUBLISH_SCHEDULE] = original[PUBLISH_SCHEDULE]
-        updates[SCHEDULE_SETTINGS] = original[SCHEDULE_SETTINGS]
```

Comment:
I think we want to unset it, but only if that item was auto published

Reply:
I see, but how can be identify item auto published.

Reply:
I think we set `auto_publish=True` on those

Reply:
okay let me check.

Reply:
I think you have to change it not only on the picture item itself but also in the associations of the text item